### PR TITLE
lsp-plugins: 1.2.1 -> 1.2.2

### PR DESCRIPTION
**Pull request by sb-child**
###### Description of changes

changelog: https://github.com/sadko4u/lsp-plugins/releases/tag/1.2.2

###### Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [ ] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- [ ] For non-Linux: Is `sandbox = true` set in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
- [ ] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://nixos.org/manual/nixpkgs/unstable/#sec-package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://nixos.org/manual/nixpkgs/unstable/#ssec-nixos-tests-linking) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [ ] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [22.11 Release Notes (or backporting 22.05 Release notes)](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md#generating-2211-release-notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
  - [ ] (Release notes changes) Ran `nixos/doc/manual/md-to-db.sh` to update generated release notes
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/applications/audio/lsp-plugins/default.nix [hunk 1]
```diff
@@ -5,11 +5,11 @@
 
 stdenv.mkDerivation rec {
   pname = "lsp-plugins";
-  version = "1.2.1";
+  version = "1.2.2";
 
   src = fetchurl {
     url = "https://github.com/sadko4u/${pname}/releases/download/${version}/${pname}-src-${version}.tar.gz";
-    sha256 = "sha256-wHibZJbrgy7t0z2rRDe1FUAG38BW/dR0JgoKVWYCn60=";
+    sha256 = "0g1q31wkxb9w792ip9qlh0gvq8gz80c3ssi2zd3pzmp6ikk7k8wi";
```
Comment (marsam) [inline pkgs/applications/audio/lsp-plugins/default.nix:12]:
```suggestion
    sha256 = "sha256-qIakDWNs8fQmlw/VHwTET2LmIvI+6I6zK88bmsWF4VI=";
```

**Review by magnetophon — APPROVED** [review]

**Review by magnetophon — CHANGES_REQUESTED** [review]
I just built this again, and it needed ``libXrandr`` as a buildInput.
No Idea if something changed since my first review or if @sb-child and me just happened to make the same mistake.

**Comment by SuperSandro2000** [comment]
FTB

```
main/x11/X11Display.cpp:43:10: fatal error: X11/extensions/Xrandr.h: No such file or directory
   43 | #include <X11/extensions/Xrandr.h>
      |          ^~~~~~~~~~~~~~~~~~~~~~~~~
compilation terminated.
```

**Comment by magnetophon** [comment]
@SuperSandro2000 fixed in https://github.com/NixOS/nixpkgs/pull/185401

**Comment by sb-child** [comment]
seems another pr has created, i will close this

**Pull request closed without merging**